doc/usage/configuration.rst

```diff
@@ -307,8 +307,8 @@ General configuration
    .. index:: default; role
 
    The name of a reST role (builtin or Sphinx extension) to use as the default
-   role, that is, for text marked up ```like this```.  This can be set to
-   ``'py:obj'`` to make ```filter``` a cross-reference to the Python function
+   role, that is, for text marked up ``like this``.  This can be set to
```

Comment:
Here, we really need the triple backquotes. Because you want to show <code>\`like this\`</code> and not just <code>like this</code>.

Reply:
Nvm, you fixed it at the same time.